src/ToDo.Skia.WPF.Host/ToDo.Skia.Wpf.Host.csproj

```diff
@@ -8,6 +8,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Identity.Client" Version="4.43.1" />
     <PackageReference Include="Refit" Version="6.3.2" />
+    <PackageReference Include="Uno.Material.WinUI" Version="2.0.0-dev.185" />
```

Comment:
shouldnt be added to those projects

Reply:
yes this change can be removed.
Also `<PackageReference Include="Uno.Material.WinUI" Version="2.0.0-dev.191" />` is already in the app.